src/commands/dev.ts

```diff
@@ -261,7 +261,6 @@ export async function command(commandOptions: CommandOptions) {
     console.log(chalk.yellow('! updating dependencies...'));
     await installCommand(commandOptions);
     await updateLockfileHash(DEV_DEPENDENCIES_DIR);
-    serverStart = Date.now();
```

Comment:
I had to wait for an install, but then it told me `Server started in 1ms!`. Now it just says "Server started."